nntrainer/include/bn_layer.h

```diff
@@ -121,10 +121,15 @@ class BatchNormalizationLayer : public Layer {
 private:
   Tensor weight;
   Tensor bias;
+  //moving mu used for inferencing.
   Tensor mu;
+  //moving var used for inferencing.
   Tensor var;
+  //current variance used in training
```

Comment:
You may want to start using doxygen-format comments. These comments may make you confused a few months later.

Reply:
Updated. Thanks :)